ros/.config/ros2/Makefile.em

```diff
@@ -39,3 +42,17 @@ ci_buildx:
 		docker push \
 			osrf/ros:$release_name-desktop; \
 	fi
+	@docker buildx build --pull --push \
```

Comment:
@ruffsl I'm not familiar with this part. is this the right way to do this ?

Reply:
This looks fine 👍